animal-trading-cards/style.css

```diff
@@ -0,0 +1,79 @@
+/* Box Model Hack */
+* {
+     box-sizing: border-box;
+}
+
+/* Clear fix hack */
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+	clear: both;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
```

Comment:
what's the change

Reply:
Added the following lines to pre-made template

https://github.com/sebz996/front-end-web-developer-course/pull/1/files#diff-48ddfca73e95f50ab8388e1f15a5f5b0df81d0137a98edcba12e00dbba71d961R24-R58